src/bootstrap/sanity.rs

```diff
@@ -169,6 +169,19 @@ pub fn check(build: &mut Build) {
             panic!("the iOS target is only supported on macOS");
         }
 
+        if target.starts_with("thumbv") {
```

Comment:
How about `contains("-none-")` because that's the part that makes them no_std?

Reply:
Done